src/docker/compose.database.yaml

```diff
@@ -20,6 +20,12 @@ services:
     build:
       context: ../../
       dockerfile: src/docker/database/Dockerfile.postgres
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready", "-d", "db_prod"]
+      interval: 30s
+      timeout: 60s
+      retries: 5
+      start_period: 80s
```

Comment:
Remove this part as it is already present in main.

Reply:
Removed in 3ef3cc9

Noting that the error motivating this duplicate healthcheck was likely a result of no recompiling the postgres image